pre-post.json

```diff
@@ -0,0 +1,16 @@
+{
+    "DestinationType" : "virtual",
+    "PartitionLayout" : [ { "disk" : "test.img", "partition" : 1, "size" : "64M", "type" : "EFI" },
+                          { "disk" : "test.img", "partition" : 2, "size" : "2G", "type" : "linux" } ],
+    "FilesystemTypes" : [ { "disk" : "test.img", "partition" : 1, "type" : "vfat" },
+                          { "disk" : "test.img", "partition" : 2, "type" : "ext4" } ],
+    "PartitionMountPoints" : [ { "disk" : "test.img", "partition" : 1, "mount" : "/boot" },
+			       { "disk" : "test.img", "partition" : 2, "mount" : "/" } ],
+    "Version": "latest",
+    "Bundles": ["os-core"],
+    "PreInstallShell": ["echo preinstall shell command"],
+    "PostNonChroot": ["./post-non-chroot.sh"],
+    "PostNonChrootShell": ["touch $ISTER_CHROOT/post-non-chroot-shell"],
+    "PostChroot": ["/post-chroot.sh"],
```

Comment:
should there be a leading dot here?
```
"PostChroot": ["./post-chroot.sh"],
```
Or not since this is in the chroot?

Reply:
Oh wait sorry I misread your quote for the current code. There shouldn't be a dot because it is the chroot.

Reply:
Right OK, +1 here then.